java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobValidatedStatus.java

```diff
@@ -16,6 +16,14 @@
 
 package sleeper.ingest.job.status;
 
+/**
+ * An interface for an ingest job info update that has validation information about the job.
```

Comment:
We can omit "An interface" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html